extension/chrome/settings/setup.htm

```diff
@@ -217,6 +228,17 @@ <h1></h1>
             </label>
           </div>
 
+          <div class="line left">
```

Comment:
When importing a key, you are working with a key that was already created before. Therefore there is no reason to ask the user what type of key they want - we're not creating anything.

Reply:
Yup, that was the dummy mistake I was suspecting. Thanks!